graphql_base/views/graphiql.xml

```diff
@@ -28,18 +28,23 @@
                 If you do not want to rely on a CDN, you can host these files locally or
                 include them directly in your favored resource bunder.
                 -->
+                <!-- GraphiQL 4.1.0 + peer-deps from a CDN -->
                 <link
-                    href="//cdn.jsdelivr.net/npm/graphiql@0.12.0/graphiql.css"
                     rel="stylesheet"
+                    href="//unpkg.com/graphiql@4.1.0/graphiql.min.css"
                 />
-                <script src="//cdn.jsdelivr.net/npm/whatwg-fetch@2.0.3/fetch.min.js" />
-                <script
-                    src="//cdn.jsdelivr.net/npm/react@16.2.0/umd/react.production.min.js"
-                />
+
+                <!-- Promise/fetch polyfills if you still need them -->
+                <script src="//cdn.jsdelivr.net/npm/whatwg-fetch@3/fetch.umd.min.js" />
+
+                <!-- React 18 UMD builds (any 16-18 will do) -->
+                <script src="//unpkg.com/react@18/umd/react.production.min.js" />
                 <script
-                    src="//cdn.jsdelivr.net/npm/react-dom@16.2.0/umd/react-dom.production.min.js"
+                    src="//unpkg.com/react-dom@18/umd/react-dom.production.min.js"
```

Comment:
Why changing the cdn, and why not using the same CDN for all parts?

In any case I'd prefer to do this in a separate PR as this is not related to the Odoo migration, and could be backported independently.